.devcontainer/devcontainer.json

```diff
@@ -5,7 +5,7 @@
   "build": {
     "dockerfile": "Dockerfile",
     "args": {
-      "VARIANT": "hirsute"
+      "VARIANT": "jammy"
```

Comment:
For some reason, some packages seem to be missing for hirsute. Switching to jammy solved the problem.

https://github.com/mizdra/eslint-interactive/runs/7589278077?check_suite_focus=true
```
#8 2.696 Err:5 http://security.ubuntu.com/ubuntu hirsute-security Release
#8 2.696   404  Not Found [IP: 185.125.190.39 80]
#8 2.730 Ign:6 http://archive.ubuntu.com/ubuntu hirsute-backports InRelease
#8 2.818 Err:7 http://archive.ubuntu.com/ubuntu hirsute Release
#8 2.818   404  Not Found [IP: 185.125.190.39 80]
#8 2.896 Err:8 http://archive.ubuntu.com/ubuntu hirsute-updates Release
#8 2.896   404  Not Found [IP: 185.125.190.39 80]
#8 2.899 Get:9 http://ppa.launchpad.net/git-core/ppa/ubuntu hirsute/main Sources [1037 B]
#8 2.975 Err:10 http://archive.ubuntu.com/ubuntu hirsute-backports Release
#8 2.975   404  Not Found [IP: 185.125.190.39 80]
#8 3.074 Get:11 http://ppa.launchpad.net/git-core/ppa/ubuntu hirsute/main amd64 Packages [3171 B]
#8 3.083 Reading package lists...
#8 3.097 E: The repository 'http://security.ubuntu.com/ubuntu hirsute-security Release' does not have a Release file.
#8 3.097 E: The repository 'http://archive.ubuntu.com/ubuntu hirsute Release' does not have a Release file.
#8 3.098 E: The repository 'http://archive.ubuntu.com/ubuntu hirsute-updates Release' does not have a Release file.
#8 3.098 E: The repository 'http://archive.ubuntu.com/ubuntu hirsute-backports Release' does not have a Release file.
#8 ERROR: executor failed running [/bin/sh -c cd /tmp/build-features/local-cache && chmod +x ./install.sh && ./install.sh]: exit code: 100
```